tests/integration/suite/actors/healthz/endpoint/noentities.go

```diff
@@ -125,10 +135,17 @@ func (n *noentities) Run(t *testing.T, ctx context.Context) {
 
 	client := util.HTTPClient(t)
 
-	req, err := http.NewRequestWithContext(ctx, http.MethodPost, fooActorURL(n.daprd), nil)
-	require.NoError(t, err)
-	resp, err := client.Do(req)
-	require.NoError(t, err)
-	assert.Equal(t, http.StatusOK, resp.StatusCode)
-	require.NoError(t, resp.Body.Close())
+	assert.EventuallyWithT(t, func(c *assert.CollectT) {
```

Comment:
We shouldn't _have_ to eventually here as a request should always succeed in creating the actor after daprd reports placement ready

Reply:
Reverted. It was part of the troubleshooting. Thanks.